src/imports/Settings/SettingsPage.qml

```diff
@@ -69,7 +69,7 @@ Page {
                 }
                 model: myself
                 delegate: ListItem.Subtitled {
-                   text:  i18n.tr("My phone number: %1").arg(phoneNumber)
+                   text:  phoneNumber.length > 0 ? i18n.tr("My phone number: %1").arg(phoneNumber) : i18n.tr("Could not read Phone Number from SIM card")
```

Comment:
Small quibble, but I think `Phone Number` should not be capitalized here, as it's not in `My phone number:` either.

Reply:
oh okay, well i forgot translation update with the last commit, will do if i don't forget it